### PR TITLE
Adds download page

### PR DESCRIPTION
**Pull request by RUGMJ**
image

### CODE REVIEW

**Comment by RUGMJ** [comment]
I'll add the icons to other sites in a different pr

**Comment by Matthias1590** [comment]
Might be nice to put the scss into a separate file instead of in the html

**Comment by Matthias1590** [comment]
Also it might just be me but I feel like its unclear that the download buttons are buttons because they are so big and have a title and description, they feel more like cards to me

File: src/pages/download.astro [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+						Download the source code and build the jar for yourself (only do
+						this if you know what you're doing)
+					</p>
+				</div>
+			</a>
+		</div>
+		<h2><a href={releaseUrl} target='_blank'>{versionNumber}</a></h2>
+		<div class='changelog' set:html={content} />
+	</main>
+	<style lang='scss' slot='head'>
```
Comment (Matthias1590) [inline src/pages/download.astro:60]: should be in a separate file so it doesnt get regenerated with every request

**Review by Matthias1590 — CHANGES_REQUESTED** [review]

File: src/pages/download.astro (hunk 1 above)
Comment (RUGMJ) [reply src/pages/download.astro:60]: how would putting it in a separate file fix that?

**Comment by RUGMJ** [comment]
> Also it might just be me but I feel like its unclear that the download buttons are buttons because they are so big and have a title and description, they feel more like cards to me

When the cursor goes over them it changes to a pointer its obvious that its a button but I see what you mean, not really sure if theres a way to fix that

**Comment by RUGMJ** [comment]
> Might be nice to put the scss into a separate file instead of in the html

The main styles are theres not a lot of scss going on here to it would be harder to maintain if it was in a separate file

File: src/pages/download.astro (hunk 1 above)
Comment (Matthias1590) [reply src/pages/download.astro:60]: cause then you can compile the scss beforehand, its also more organized

Comment (RUGMJ) [reply src/pages/download.astro:60]: the scss is pre compiled, I find it easier to work on a page when the scss is in the same file plus it keeps the file structure clean

**Comment by Matthias1590** [comment]
> > Also it might just be me but I feel like its unclear that the download buttons are buttons because they are so big and have a title and description, they feel more like cards to me
> 
> When the cursor goes over them it changes to a pointer its obvious that its a button but I see what you mean, not really sure if theres a way to fix that

the buttons should probably be smaller, maybe we can follow the md guidelines here so they feel more like buttons?

File: src/pages/download.astro (hunk 1 above)
Comment (Matthias1590) [reply src/pages/download.astro:60]: if im looking for css i dont expect it to be in an html file, having a separate file for css does not make the file structure messy as long as you keep things separated and dont throw everything into a single folder

**Comment by RUGMJ** [comment]
If theyre smaller the page would look empty, I think they're clear enough to be buttons because of the command words used in the descriptions

File: src/pages/download.astro (hunk 1 above)
Comment (RUGMJ) [reply src/pages/download.astro:60]: This isnt a html file its an astro file which contains the content of the page, I see what you mean and normally I would agree but theres so little css specific to the page that I dont think it warrants its own file

**Comment by Matthias1590** [comment]
> If theyre smaller the page would look empty, I think they're clear enough to be buttons because of the command words used in the descriptions

I think it'd look better if there were 4 normal sized buttons than 4 buttons taking up the entire screen though, even if it might look a little empty but ig this works

File: src/pages/download.astro (hunk 1 above)
Comment (Matthias1590) [reply src/pages/download.astro:60]: still think its nice to keep things clean from the very beginning so things dont get messy later on, guess it works though

**Pull request merged**